src/lumigo_tracer/tracer.py

```diff
@@ -66,7 +66,8 @@ def lambda_wrapper(*args, **kwargs):
                     SpansContainer.get_span().add_exception_event(e, inspect.trace())
                 raise
             finally:
-                SpansContainer.get_span().end(ret_val)
+                event = args[0] if len(args) > 0 else None
```

Comment:
this logic should not be here.
`SpansContainer` should remember the event by itself